File: GTTS/GTTS.Web/ClientApp/src/app/models/person-attachment.ts

export class PersonAttachment 
{
    public PersonID: number = 0;
    public FileID: number = 0;
    public FileName: string = "";
    public FileLocation: string = "";
    public PersonAttachmentTypeID: number = 0;
    public PersonAttachmentType: string = "";
    public Description: string = "";
    public IsDeleted?: boolean;
    public ModifiedByAppUserID: number = 0;
    public FullName: string = "";
    public ModifiedDate: Date = new Date(0);
}

File: GTTS/INL.PersonService.Models/PersonAttachment.cs
using System;

namespace INL.PersonService.Models
{
    public class PersonAttachment : IPersonAttachment
    {
        public long PersonID { get; set; }
        public int FileID { get; set; }
        public string FileName { get; set; }
        public string FileLocation { get; set; }
        public int PersonAttachmentTypeID { get; set; }
        public string PersonAttachmentType { get; set; }
        public string Description { get; set; }
        public bool? IsDeleted { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public string FullName { get; set; }
        public DateTime ModifiedDate { get; set; }
        public byte[] FileContent { get; set; }
        public byte[] FileHash { get; set; }
        public int FileSize { get; set; }
    }
}
